flink-ml-iteration/src/main/java/org/apache/flink/iteration/Iterations.java

```diff
@@ -117,19 +117,16 @@
      */
     public static DataStreamList iterateUnboundedStreams(
             DataStreamList initVariableStreams, DataStreamList dataStreams, IterationBody body) {
-        return createIteration(
+        return iterateStreams(
```

Comment:
After this renaming, having `iterateUnboundedStreams` and `iterateStreams`, without `iterateBoundedStreamsUntilTermination` seems a bit confusing to me. These APIs are not symmetric for bounded and unbounded cases. A side effect of doing so is that we cannot restrict the following use case: supporting termination criteria on iterating unbounded streams.


Could you explain why do you propose to make the change here?

Reply:
This change is reverted.